app/assets/stylesheets/users.scss

```diff
@@ -75,15 +87,85 @@
   width: 100%;
 }
 
-.users-social-links {
-  height: 50px;
-  margin-left: 5px;
-  margin-right: 5px;
-  padding-top: 10px;
-  width: 40px;
+
+.auth-btns-container {
+  display: flex;
+  flex-wrap: wrap;
+  height: 120px;
+}
+
+// sign in button styles
+.auth-btn {
+  border-radius: 2px;
+  box-shadow: 0 3px 4px 0 $auth-btn-shadow;
+  height: 42px;
```

Comment:
Pattern for the class name prefixes is to same as stylesheet name. Use something more precise like users-social-button. Also, we already have a variable for box-shadow; use that to keep everything consistent